applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -1674,6 +1731,34 @@ else if ( maximumAdContentRating == 3 )
         return AppLovinAdContentRating.NONE;
     }
 
+    private static ConsentFlowUserGeography getAppLovinConsentFlowUserGeography(final String userGeography)
+    {
+        if ( "G".equalsIgnoreCase( userGeography ) )
```

Comment:
nit: Let's extract constants for `"G"`/`"O"`, since we are using them in couple of different places.